### PR TITLE
Add warning about receiving DTMF on iOS devices in data calling

### PR DESCRIPTION
**Pull request by GiovanniBeriSinch**

### CODE REVIEW

File: docs/voice/voice-for-android-cloud/voice-android-cloud-calling.md [hunk 1]
```diff
@@ -39,6 +39,15 @@ Placing an _app-to-phone_ call requires a developer account with credits. Toppin
 
 _App-to-phone_ calls can be tested by calling the following test number: _+46000000000_. When placing a call to this number, you will hear a voice prompt stating that the call has been connected, and shortly after that the call will automatically be ended.
 
+> ⚠ Credits Required
```
Comment (GiovanniBeriSinch) [inline docs/voice/voice-for-android-cloud/voice-android-cloud-calling.md:42]: I copy-pasted this from correspondent iOS page, I guess it should be here as well

File: docs/voice/voice-for-android-cloud/voice-android-cloud-calling.md [hunk 2]
```diff
@@ -39,6 +39,15 @@ Placing an _app-to-phone_ call requires a developer account with credits. Toppin
...
+>
+> Placing an _App-to-Phone_ call requires that your Sinch account has sufficient credits; Top up credits on your [Account](https://portal.sinch.com/#/account) page. Credits are used each time an _App-to-Phone_ call is placed and your account balance is updated after each call.
+
+> ⚠ Testing DTMF
+>
+> In _App-to-Phone_ scenario, it is possible to issue DTMF sequences using Sinch SDK. Please note that if the receiving end of the call is an iOS device, you might have to disable _VoLTE_ ("Voice over LTE") option in the phone at the receiving end of the call in order to be able to hear DTMF tones.
```
Comment (dmipon) [inline docs/voice/voice-for-android-cloud/voice-android-cloud-calling.md:48]: in the phone **settings** at

I think it's better.

File: docs/voice/voice-ios-cloud/voice-ios-cloud-calling.md [hunk 3]
```diff
@@ -46,6 +46,10 @@ _App-to-Phone_ calls can be tested by calling the following test number: _+46000
 >
 > Placing an _App-to-Phone_ call requires that your Sinch account has sufficient credits; Top up credits on your [Account](https://portal.sinch.com/#/account) page. Credits are used each time an _App-to-Phone_ call is placed and your account balance is updated after each call.
 
+> ⚠ Testing DTMF
+>
+> In _App-to-Phone_ scenario, it is possible to issue DTMF sequences using Sinch SDK. Please note that if the receiving end of the call is an iOS device, you might have to disable _VoLTE_ ("Voice over LTE") option in the phone at the receiving end of the call in order to be able to hear DTMF tones.
```
Comment (dmipon) [inline docs/voice/voice-ios-cloud/voice-ios-cloud-calling.md:51]: Same if you agree

**Comment by dmipon** [comment]
LGTM

File: docs/voice/voice-for-android-cloud/voice-android-cloud-calling.md (hunk 2 above)
Comment (GiovanniBeriSinch) [reply docs/voice/voice-for-android-cloud/voice-android-cloud-calling.md]: I changed it to:
_"disable [...] option **in the settings of the phone** at the receiving end of the call"_

**Pull request merged**